patroni/dcs/kubernetes.py

```diff
@@ -1325,19 +1327,27 @@ def touch_member(self, data: Dict[str, Any]) -> bool:
             role = None
             tmp_role = None
 
-        role_labels = {self._role_label: role}
+        updated_labels = {self._role_label: role}
```

Comment:
isn't the name change a bit too generic?

Reply:
it is about different labels, thus the name